### DISCUSSION TITLE
Specifying buyerOrderDate results in a [CII-DT-027] error

### DISCUSSION THREAD

**Discussion by AxelStallknecht** [stephanstapel/ZUGFeRD-csharp#321] [General]
Hello Stephan, if i provide the buyerOrderDate I get an [CII-DT-027] error. The complete error message when I check the resulting file is:
<html>
<body>

[CII-DT-027] - FormattedIssueDateTime should not be present
--
Pfad: /rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction[1]/ram:IncludedSupplyChainTradeLineItem[1]/ram:SpecifiedLineTradeAgreement[1]/ram:BuyerOrderReferencedDocument[1]

</body>
</html>

I checked the resulting file under https://erechnungsvalidator.service-bw.de/. I have no idea what can be wrong. Any suggestions?

BR
Axel

**Comment by stephanstapel** [comment]
looks like this should only be written in Extended profile:

![grafik](https://github.com/user-attachments/assets/41d32b5c-5d39-4f77-8434-b19bdc5aaaf7)

just asking myself why one would write a BuyerOrderReferencedDocument without date and issuer assigned id but whatever, I fixed it:
https://github.com/stephanstapel/ZUGFeRD-csharp/commit/c5ce80dfba5db4305726683023126c6f65a81841
